jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/FileSystemPool.java

```diff
@@ -164,7 +171,23 @@ private void unmount(URI fsUri)
             {
                 if (LOG.isDebugEnabled())
                     LOG.debug("Ref counter reached 0, closing pooled FS {}", bucket);
-                IO.close(bucket.fileSystem);
+                try
+                {
```

Comment:
Good try, but I agree it would be nicer if we could avoid this. See my comment https://github.com/jetty/jetty.project/issues/11548#issuecomment-2014626118

Reply:
IMHO we should add this workaround on top of properly ordering our file deletion & FS closing or we might get bitten again in the future.